py/torch_tensorrt/dynamo/_settings.py

```diff
@@ -74,7 +73,7 @@ class CompilationSettings:
         output_format (str): Output format of the result of TRT compilation. Options include "exported_program" (or) "ep" | "torchscript" (or) "ts" | "graph_module" (or) "fx". Default is "exported_program"
     """
 
-    precision: torch.dtype = PRECISION
+    precision: dtype = field(default_factory=lambda: PRECISION)
```

Comment:
What is the reasoning for using the `field` here?

Reply:
Its a WAR dealing with enums not being immutable